src/components/sbb-slider/sbb-slider.tsx

```diff
@@ -117,12 +121,13 @@ export class SbbSlider implements ComponentInterface {
     const mathFraction: number = (value - min) / (max - min);
     this._valueFraction =
       isNaN(mathFraction) || mathFraction < 0 ? 0 : mathFraction > 1 ? 1 : mathFraction;
-    this._syncValues();
+    this._syncValues(value);
   }
 
-  @Listen('keydown')
-  public async handleKeydown(event: KeyboardEvent): Promise<void> {
-    if (this.disabled || this.readonly || (await isEventPrevented(event))) {
+  private async _handleKeydown(event: KeyboardEvent): Promise<void> {
+    // FIXME the third condition in the if doesn't prevent from scrolling
+    event.preventDefault();
```

Comment:
Don't we change the logic here?

Reply:
Marking as resolved, but discussion thread is still open in the other comment https://github.com/lyne-design-system/lyne-components/pull/2082#discussion_r1362142375